doc/releases/release-notes-4.0.rst

```diff
@@ -210,6 +213,7 @@ Boards & SoC Support
    * :ref:`Renesas RA8T1 Evaluation Kit <mcb_ra8t1>` (``mck_ra8t1``)
    * :zephyr:board:`Renode Cortex-R8 Virtual <cortex_r8_virtual>` (``cortex_r8_virtual``)
    * :ref:`sensry.io Ganymed Break-Out-Board (BOB) <ganymed_bob>` (``ganymed_bob``)
+   * :zephyr:board:`Seeed XIAO ESP32-S3 Sense Variant <xiao_esp32s3>`: ``xiao_esp32s3``.
```

Comment:
@sylvioalves 
Sorry that the error log is not more friendly :/

```suggestion
   * :ref:`Seeed XIAO ESP32-S3 Sense Variant <xiao_esp32s3>`: ``xiao_esp32s3``.
```

another option would be to fix boards/seeed/xiao_esp32s3/doc/index.rst so that it's migrated to `zephyr:board` directive (I think it felt trough the cracks as the PR introducing the variant has overriden the changes I had made in b58b8964d3d3a3833d04786113da38f76d53953f)

File should start with:

```
.. zephyr:board:: xiao_esp32s3

Overview
********

Seeed Studio XIAO ESP32S3 and XIAO ESP32S3 Sense are IoT mini development boards based on the
Espressif ESP32-S3 WiFi/Bluetooth dual-mode chip.

For more details see the `Seeed Studio XIAO ESP32S3`_ wiki page.

.. figure:: img/xiao_esp32s3.jpg
   :align: center

..........
```